gate-web/gate-web.gradle

```diff
@@ -35,6 +35,9 @@ dependencies {
   compile('org.apache.commons:commons-io:1.3.2')
 
   compile('org.springframework.session:spring-session-data-redis:1.1.1.RELEASE')
+  compile "org.springframework.data:spring-data-redis:1.8.10.RELEASE"
+  compile "org.springframework.session:spring-session:1.3.1.RELEASE"
+  compile "redis.clients:jedis:2.9.0"
```

Comment:
Could we do `compile spinnaker.dependency("jedis")` here?

Reply:
Done.